src/subscription/checkout/StripeOptions.js

```diff
@@ -10,8 +10,10 @@ export const getStripeOptions = ({ currency, price }) => ({
    * NOTE: this is important to set the right subscription price as defined within stripe subscription
    * otherwise card details will not load properly
    * https://dashboard.stripe.com/test/subscriptions/sub_1Mtd3sH4caH7G0X1dvzlMT3o
+   * Stripe consider amount in cents and minimum amount it expects is 0.50 cents
```

Comment:
Can you change this to `* Stripe expects the amount in cents for USD currency, and minimum amount it expects is 0.50 cents`

Reply:
We should ask QA to test this in localized currencies. BE sends the price in USD, so I think we're good here but just in case

Reply:
done.